CHANGELOG.md

```diff
@@ -2,6 +2,30 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [3.0.0](https://github.com/fiatconnect/fiatconnect-types/compare/v1.0.0...v3.0.0) (2022-04-25)
```

Comment:
Hmmm, this looks like maybe a bug with our standard-version configuration. It should be comparing 2.1.1 not 1.0.0 🤔

Reply:
doh

Reply:
Some of the changes below are from before 2.1.1